Modify the config-clang-mac introduction, so it compiles properly and… (#6467)
microsoft/vscode-docs: docs/cpp/config-clang-mac.md

   ![Build task](images/clang-mac/default-build-task.png)

> You'll only be asked to choose a compiler the first time you run `helloworld.cpp`. This compiler will be set as the "default" compiler in `tasks.json` file. Since we are using features from C++17 in this example, but the default C++ version in macOS is C++98, the first time you attempt to build, it will fail with Error "expected ';' at end of declaration gcc [Ln 9, Col 23]".

Reviewer: ```suggestion
You'll only be asked to choose a compiler the first time you run `helloworld.cpp`. This compiler will be set as the "default" compiler in `tasks.json` file. Because this example uses C++17, but the default C++ version in macOS is C++98, the first build attempt will fail with Error `expected ';' at end of declaration gcc [Ln 9, Col 23]`.
```

wouldn't it be better code formatting here?

4. Make sure the C++17 standard is used in this Workspace.

> The first time you run your program, the C++ extension creates `tasks.json`, which you'll find in your project's `.vscode` folder. `tasks.json` stores build configurations. Here you have to add the following line: `"-std=c++17",` as the first  argument passed to clang++.

Reviewer: ```suggestion
The first time you run your program, the C++ extension creates `tasks.json`, which you'll find in your project's `.vscode` folder. `tasks.json` stores build configurations. Add the following line as the first argument passed to clang++ here: `"-std=c++17",`.
```

```json
    "group": "build",
```
5. After the build succeeds, your program's output will appear in the integrated **Debug Console**.
    ![screenshot of debug console output](images/clang-mac/helloworld-debug-console-output.png)

> 

Reviewer 2: build up with software

> If you would like it to show up in the terminal, you have to tell the task to also **run** the generated program, by adding `"&&",` and `"${fileDirname}/${fileBasenameNoExtension}"` to the end of the list of arguments in `tasks.json`.

Reviewer: ```suggestion
To show it in the terminal, add`"&&",` and `"${fileDirname}/${fileBasenameNoExtension}"` to the end of the arguments list in the `tasks.json` file. This will **run** the generated program in the task.
```
